plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -551,23 +551,64 @@ private function add_organization_group_blocks() {
 			)
 		);
 		// Attributes section.
-		$product_catalog_section = $organization_group->add_section(
+		$product_attributes_section = $organization_group->add_section(
 			array(
 				'id'         => 'product-attributes-section',
 				'order'      => 20,
 				'attributes' => array(
 					'title'       => __( 'Attributes', 'woocommerce' ),
 					'description' => __( 'Add descriptive pieces of information that customers can use to filter and search for this product. <a href="https://woo.com/document/managing-product-taxonomies/#product-attributes" target="_blank" rel="noreferrer">Learn more</a>.', 'woocommerce' ),
+					'blockGap'    => 'unit-40',
 				),
 			)
 		);
-		$product_catalog_section->add_block(
+		$product_attributes_section->add_block(
 			array(
 				'id'        => 'product-attributes',
 				'blockName' => 'woocommerce/product-attributes-field',
 				'order'     => 10,
 			)
 		);
+
+		if ( Features::is_enabled( 'product-custom-fields' ) ) {
+			$organization_group->add_section(
+				array(
+					'id'    => 'product-custom-fields-wrapper-section',
+					'order' => 30,
+				)
+			)->add_block(
+				array(
+					'id'         => 'product-custom-fields-toggle',
+					'blockName'  => 'woocommerce/product-custom-fields-toggle-field',
+					'order'      => 10,
+					'attributes' => array(
+						'label' => __( 'Show custom fields', 'woocommerce' ),
+					),
+				)
+			)->add_block(
+				array(
+					'id'         => 'product-custom-fields-section',
+					'blockName'  => 'woocommerce/product-section',
+					'order'      => 10,
+					'attributes' => array(
+						'blockGap'    => 'unit-30',
+						'title'       => __( 'Custom fields', 'woocommerce' ),
+						'description' => sprintf(
+							/* translators: %1$s: Custom fields guide link opening tag. %2$s: Custom fields guide link closing tag. */
+							__( 'Custom fields can be used in a variety of ways, such as sharing more detailed product information, showing more input fields, or internal inventory organization. %1$sRead more about custom fields?%2$s', 'woocommerce' ),
+							'<a href="https://woo.com" target="_blank" rel="noreferrer">',
```

Comment:
The link in the design is pointing to `woo.com` but I think that it would be better to point to [the WordPress custom fields documentation](https://wordpress.org/documentation/article/assign-custom-fields/).

Reply:
Yes, agree! That's the correct link, thank you!